packages/dataviews/src/components/dataviews-filters/style.scss

```diff
@@ -355,12 +355,16 @@
 	.dataviews-filters__search-widget-filter-combobox__icon {
 		position: absolute;
 		inset-inline-start: $grid-unit-15;
-		top: 50%;
-		transform: translateY(-50%);
+		top: 0;
+		bottom: 0;
 		display: flex;
 		align-items: center;
 		justify-content: center;
 		width: $icon-size;
+
+		&:dir(ltr) {
+			transform: scaleX(-1);
+		}
```

Comment:
<img width="340" alt="Search widget in DataViews" src="https://github.com/user-attachments/assets/9f53fefd-6bde-4763-9fed-3e8bdc9a4244" />
<img width="340" alt="Search widget in DataViews, RTL" src="https://github.com/user-attachments/assets/1108da3b-d783-49a0-b628-49db05ada1aa" />